sedaro/src/sedaro/results/series.py

```diff
@@ -118,12 +149,12 @@ def plot(self, show=True, ylabel=None, elapsed_time=True, height=None, xlim=None
     def __plot(self, show, ylabel, elapsed_time, height, xlim, ylim, **kwargs):
         if not PLOTTING_ENABLED:
             raise ValueError('Plotting is disabled because matplotlib could not be imported.')
-        if self.__has_subseries:
+        if self.__has_subseries and not self.__is_singleton_or_vector():
             raise ValueError('Select a specific subseries to plot.')
         try:
             if height is not None:
                 plt.rcParams['figure.figsize'] = [plt.rcParams['figure.figsize'][0], height]
-            plt.plot((self.__elapsed_time if elapsed_time else self.__mjd), self.__series, **kwargs)
+            plt.plot((self.__elapsed_time if elapsed_time else self.__mjd), self.values, **kwargs)
```

Comment:
I need to understand how this works in the `__is_singleton_or_vector` truthy case.

Reply:
Double check multi-dimensional "vector" here as well.